drivers/dma/dma_stm32.c

```diff
@@ -30,7 +30,7 @@ LOG_MODULE_REGISTER(dma_stm32, CONFIG_DMA_LOG_LEVEL);
 #endif
 
 #if DT_NODE_HAS_STATUS_OKAY(DT_DRV_INST(0))
-#if DT_INST_IRQ_HAS_IDX(0, 7)
+#if DT_INST_IRQ_HAS_IDX(0, 7) || defined(CONFIG_SOC_SERIES_STM32WB0X)
```

Comment:
Some information (could be in commit message) about why a `CONFIG_SOC_SERIES_STM32WB0X` specific rule is needed would be welcome.

Reply:
Added a comment above.